Model/AbstractApi.php

```diff
@@ -149,6 +174,15 @@ public function sendApiRequest($path, array $data, $method = "post", $timeout =
                 $data
             );
 
+            if (($this->_getCurlStatus() == 401) && (self::$_retryCount < self::NUMBER_OF_RETRY)) {
+                $this->_yotpoConfig->log("AbstractApi::sendApiRequest() - response: ", "debug", $this->_prepareCurlResponseData());
+                $this->resourceConfig->deleteConfig(YotpoConfig::XML_PATH_YOTPO_TOKEN, 'stores', $this->_yotpoConfig->getCurrentStoreId());
+                $token = $this->oauthAuthentication($this->_yotpoConfig->getCurrentStoreId());
```

Comment:
i think you want to force create a token here no?
$this->oauthAuthentication($this->_yotpoConfig->getCurrentStoreId(), **true**)

Reply:
Yes, your right. Thanks for pointing this. Apologies for missing this out. Will do the desired changes.